projects/ontimize-web-ngx/src/lib/components/table/extensions/footer/paginator/o-table-paginator.component.ts

```diff
@@ -33,12 +29,13 @@ export class OTablePaginatorComponent extends OBaseTablePaginator implements OTa
     @Inject(forwardRef(() => OTableComponent)) protected table: OTableComponent
   ) {
     super();
+  }
+
+  ngOnInit() {
     this.pageSize = this.table.queryRows;
```

Comment:
Instead of pointing to _this.table.queryRows_ I think it would be better to use the table method _  get pageLenght(): number_ what do you think?

Reply:
clarified with Manuel. Everything is fine